source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Continuous Integration with CircleCI
+description: How to integrate Pantheon with CircleCI and GitHub.
+tags: [siteintegrations]
+type: guide
+permalink: docs/guides/:basename/
+date: 4/25/2017
+contributors: [scottmassey]
+---
+
+Continuous Integration service providers such as [CircleCI](https://circleci.com) allow developers to automate the process of building, testing, and deploying code. Given sufficient test coverage, continuous integration ensures errors are caught before code is deployed.
+
+## What You'll Build
+- A new Drupal 8 site using an [example repository](https://github.com/pantheon-systems/example-drops-8-composer) that contains Drupal 8, [Drush](http://docs.drush.org/en/master), and [Drupal Console](https://drupalconsole.com)- all of which will be installed and managed with [Composer](https://getcomposer.org).
+- A simple implementation of a [Behat](http://behat.org/en/latest/) [test suite](https://github.com/pantheon-systems/example-drops-8-composer/tree/master/tests) and other required configuration files.
+- A new GitHub repository and continuous integration implementation on CircleCI.
+
+When code is submitted to GitHub for deployment, CircleCI will create a testing environment and run a suite of tests. If tests pass, the code can be merged into the master branch of the codebase. Before this merge is deployed, CircleCI will repeat the testing process against the production branch.
+
+
+## What You’ll Need
+
+- [Composer](https://getcomposer.org/), [Git](https://git-scm.com/download/) and [Terminus](/docs/terminus) running on your local computer.
+- A GitHub account
+- A [Multidev](/docs/multidev/) enabled site on Pantheon. Signup at [Pantheon for Agencies](https://pantheon.io/agencies/pantheon-for-agencies) (free) to enable Multidev.
```

Comment:
This is a little confusing since we create a new site in a section below. The user doesn't necessarily need a new site before they start - they really just need to have a P4A account, right?